nym-vpn-core/crates/nym-vpn-cli/src/error.rs

```diff
@@ -26,6 +26,12 @@ pub(crate) enum Error {
 
     #[error("failed to parse encoded credential data")]
     FailedToParseEncodedCredentialData(#[source] bs58::decode::Error),
+
+    #[error("{0}")]
```

Comment:
In some of your PRs you used `#[transparent]` instead. Does it not work for a boxed error?

Reply:
It only works when you have `#[from]`, which in this case I wanted to avoid